SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -144,7 +144,7 @@ + (void)asyncRegisterUnseenDatabaseView
 
     [self registerMessageDatabaseViewWithName:TSUnseenDatabaseViewExtensionName
                                  viewGrouping:viewGrouping
-                                      version:@"1"
+                                      version:@"2"
```

Comment:
As long as we're bumping the DB view, should the DB view have logic to discard anything which `isDynamicInteraction` - then we wouldn't have to worry about the "read"/"unread" status of the interactions.